ccpp/physics_namelists/input_RAP_mmm.nml

```diff
@@ -0,0 +1,136 @@
+&gfs_physics_nml
+  fhzero       = 6
+  h2o_phys     = .true.
+  ldiag3d      = .true.
+  qdiag3d      = .true.
+  print_diff_pgr = .false.
+  fhcyc        = 24
+  use_ufo      = .true.
+  pre_rad      = .false.
+  imp_physics  = 8
+  ltaerosol    = .true.
+  lradar       = .true.
+  sedi_semi    = .true.
+  decfl        = 10
+  nssl_cccn    = 0.6e9
+  nssl_alphah  = 0.0
+  nssl_alphahl = 1.0
+  nssl_hail_on = .false.
+  nssl_ccn_on  = .true.
+  nssl_invertccn = .true.
+  dt_inner     = 60
+  ttendlim     = -999
+  pdfcld       = .false.
+  fhswr        = 1200.
+  fhlwr        = 1200.
+  ialb         = 2
+  iems         = 2
+  iaer         = 5111
+  icliq_sw     = 1
+  iovr         = 1
+  icloud       = 0
+  ico2         = 2
+  isubc_sw     = 2
+  isubc_lw     = 2
+  isol         = 2
+  lwhtr        = .true.
+  swhtr        = .true.
+  cnvgwd       = .true.
+  do_deep      = .true.
+  shal_cnv     = .true.
+  cal_pre      = .false.
+  redrag       = .true.
+  dspheat      = .true.
+  hybedmf      = .false.
+  satmedmf     = .false.
+  isatmedmf    = 0
+  do_mynnedmf  = .true.
```

Comment:
if do_ysu = .true., should we make do_mynnedmf = .false.? Or, this will be overwritten wen do_ysu = .true.? Tks! @dustinswales